torchvision/models/mobilenet.py

```diff
@@ -74,12 +94,12 @@ def __init__(self, num_classes=1000, width_mult=1.0, inverted_residual_setting=N
                              "or a 4-element list, got {}".format(inverted_residual_setting))
 
         # building first layer
-        input_channel = int(input_channel * width_mult)
-        self.last_channel = int(last_channel * max(1.0, width_mult))
+        input_channel = _make_divisible(input_channel * width_mult, 8)
```

Comment:
Should we add the `8` as a parameter, potentially letting the user choose different values?
I believe the only reason they used `8` here was because the underlying kernels might be optimized for multiples of 8